src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/Icons.xaml

```diff
@@ -78,4 +78,54 @@
         <Rectangle Width="14" Height="14" RadiusX="7" RadiusY="7" Fill="{StaticResource Toggl.AccentBrush}" StrokeThickness="1"/>
         <Path Canvas.Top="3.2" Canvas.Left="2.5" Fill="{DynamicResource Toggl.CardBackground}" StrokeThickness="1" Data="M3.75914017 5.02308449 L7.00866193 0.411902318 C7.30820656 -0.0131617185 7.91527881 -0.128022547 8.36459575 0.155353478 C8.81391269 0.438729502 8.93532714 1.01303364 8.63578251 1.43809768 L4.7246714 6.98809768 C4.38240853 7.47378037 3.65601908 7.54182469 3.21971781 7.12907377 L0.77527337 4.81657377 C0.393427395 4.4553388 0.393427395 3.8696612 0.77527337 3.50842623 C1.15711934 3.14719126 1.77621399 3.14719126 2.15805996 3.50842623 L3.75914017 5.02308449 Z"/>
     </Canvas>
+    <Viewbox Stretch="Uniform" x:Key="Toggl.DatePickerLeft">

```

Comment:
I guess you just copy-pasted it from an icon converter tool so it is OK considering it's just an icon, but I find it nice that the whole thing can be simplified to:
```
    <Viewbox Stretch="Uniform" x:Key="Toggl.DatePickerLeft">
        <Path Width="24" Height="25" Fill="#FF333333" Data="M9.902 13l5.362-5.361-.528-.528L8.848 13l5.888 5.889.528-.528z" />
    </Viewbox>
```